Exercise2-Group Hinata.R

```diff
@@ -0,0 +1,21 @@
+# Install and load necessary packages
+#install.packages("gtsummary")
+#library(gtsummary)
+#library(ggplot2)
```

Comment:
Dear Ernest

I did that but every time I open the code and run it some functions don't run because it says packages not installed. That's why I had to put them again

Reply:
I ran your code locally on my computer and all just worked fine.